### DISCUSSION TITLE
Welcome to The Secretary Discussions!

### DISCUSSION THREAD

**Discussion by mikaelstaer** [mikaelstaer/The-Secretary#12] [Announcements]
🕺💃

**Comment by mikaelstaer** [comment]
Working on PHP 8 compatible updates.

**Reply by mikaelstaer** [reply]
Going well. Lots to update, PHP 8 is a huge step forward. Almost done.

**Reply by thyrrestrup** [reply]
Any update  on PHP 8 support?

**Reply by mikaelstaer** [reply]
This was delayed, but picking it up again now. Doing a test install for a current user, hopefully that goes smoothly and this can be released to the public.

**Reply by thyrrestrup** [reply]
That's awesome! Thanks for a great cms. I've used it for many years ;-)

**Reply by mikaelstaer** [reply]
That's good to hear! Hope to have this done soon...
